### ISSUE TITLE
Reproducing bbox layout experiment

### ISSUE THREAD

**Issue by ktrk115** [JiananLi2016/LayoutGAN-Tensorflow#6]
Hi, Jianan. Thanks for sharing the code.
I am trying to reproduce the bbox layout experiment in my forked repository.
Referring to the paper and the commented code, I obtained the following generated results.

Real example | Generated example
:--: | :--: 
![](https://raw.githubusercontent.com/ktrk115/LayoutGAN-Tensorflow/8460cb20b9a6c1c27db0983c455e11bd1b2f6b7b/demo/0d02d10/sample.jpg) | ![](https://raw.githubusercontent.com/ktrk115/LayoutGAN-Tensorflow/8460cb20b9a6c1c27db0983c455e11bd1b2f6b7b/demo/0d02d10/train_49_2512.jpg)

Do you think the quality of these generated results is as good as the results you got?
Do you expect improvement by adjusting hyperparameters? Which parameters in particular do you feel have a significant impact?
Also, I would like to know if there are any significant differences in my implementation of the NN architecture.

My experiment is summarized [here](https://github.com/ktrk115/LayoutGAN-Tensorflow/blob/c711214ea9a9cea1b987689c00391053eed26e43/doc_experiment.md).

**Comment by sharmasapna** [comment]
Hi , I also trying to run Jianan code for bbox layout experiment. But facing python2 cpmpatibility issues. Also data dimension is not very clear to me. Can you share your working code with dataset ?

**Comment by ktrk115** [comment]
@sharmasapna See my [repository](https://github.com/ktrk115/LayoutGAN-Tensorflow) and [report](https://github.com/ktrk115/LayoutGAN-Tensorflow/blob/c711214ea9a9cea1b987689c00391053eed26e43/doc_experiment.md).

**Comment by sharmasapna** [comment]
@ktrk115 Thanks for sharing the code. preprocess_doc.py was really helpful. Did you also work on clipart abstract scene generation? Where can I get the labeled data for the same?

**Comment by ktrk115** [comment]
@sharmasapna No, I do not work on clipart. You can get the data [here](https://vision.ece.vt.edu/clipart/), but please check the paper before asking, as the authors should have cited it in their paper.

**Comment by sharmasapna** [comment]
> here

Thanks!

**Comment by UnmeshRaskar** [comment]
@sharmasapna Hi, I am also trying to implement LayoutGAN bbox generation for RICO/Clipart scene synthesis using Kikuchi's repo. I am running the code in Colab by downgrading the tf version and for some reason, no GPU is being utilised. Also, after a few epochs, i am getting all mode collapsed samples. Did the code give you good results? Can you please suggest some changes?

**Comment by DobbieMcDobster** [comment]
Hi @UnmeshRaskar, I am likewise trying to reproduce LayoutGAN - were you able to resolve the issue? I happen to face a similar issue. Many thanks in advance!
